src/simulate.js

```diff
@@ -33,7 +33,7 @@ var launchServer = function (opts) {
     config.platform = platform;
     config.simHostOptions = simHostOpts;
     config.telemetry = opts.telemetry;
-    config.liveReload = !!opts.livereload;
+    config.liveReload = opts.hasOwnProperty('livereload') ? opts.livereload : true;
```

Comment:
Might as well, to make sure it's saved as a boolean in the config module, so I'll add it back. Thanks!